src/main/java/htsjdk/samtools/BAMFileReader.java

```diff
@@ -329,6 +329,29 @@ public ValidationStringency getValidationStringency() {
         return mCurrentIterator;
     }
 
+    /**
+     * Prepare to iterate through the SAMRecords in file order, starting at the given
+     * file pointer. The restrictions described for {@link #getIterator()} apply here.
```

Comment:
It seems like this is unused in the GATK/Hadoop-BAM client code in favor of `getIterator(final SAMFileSpan chunks)` -- can we remove?